pysteps/blending/utils.py

```diff
@@ -28,6 +28,7 @@
 from pysteps.cascade.bandpass_filters import filter_gaussian
 from pysteps.exceptions import MissingOptionalDependency
 from pysteps.utils import get_method as utils_get_method
+from pysteps.utils.check_norain import check_norain as new_check_norain
```

Comment:
Is this really necessary? If the default behaviour of `check_norain` method doesn't change with respect to the original method, it'd be enough to import it here, remove the older one, without need for a deprecation warning. Do you agree?

Reply:
you also have a good point! Let's leave it!